src/KMonad/Model/Button.hs

```diff
@@ -191,6 +194,65 @@ around outer inner = Button
   (Action (runAction (outer^.pressAction)   *> runAction (inner^.pressAction)))
   (Action (runAction (inner^.releaseAction) *> runAction (outer^.releaseAction)))
 
+-- | A variant of `around` which releases it's outter button when another
+-- key is pressed.
+aroundOnly ::
+     Button -- ^ The outer 'Button'
+  -> Button -- ^ The inner 'Button'
+  -> Button -- ^ The resulting nested 'Button'
+aroundOnly outer inner = onPress $ do
+  runAction $ outer^.pressAction
+  runAction $ inner^.pressAction
+  go =<< matchMy Release
+  pure ()
+ where
+  go :: KeyPred -> AnyK ()
+  go isMyRelease = hookF InputHook $ \e ->
+    if
+      | isMyRelease e -> do
+        runAction $ inner^.releaseAction
+        runAction $ outer^.releaseAction
+        pure Catch
+      | isPress e -> do
+        runAction $ outer^.releaseAction
```

Comment:
When having to buttons `A` and `B` and they are pressed in sequence, we have the following output
`Psft Pa Rsft Psft Pb`
Previously I thought this wasn't an issue since pressing and releasing modifier keys is mostly irrelevant except for games, but there you wouldn't want complex buttons.
But there are two modifiers for which this is only partially true: `super` and `alt` since a tap `super` may result in a menu opening and `alt` is commonly used to focus the nav bar of an application.

Reply:
I tested Firefox and libre office with regard to pressing alt another key and repressing alt.
In LibreOffice everything works as expected, since the bar is active when alt is held. In firefox it vanishes on the second press as alt toggles the bar.
Fixing this would require a special button which waits for the next event and in case it is its own release annihilates both outgoing events. This would require some changes to the kmonad core event handler.

Reply:
An `implicit-around` of some modifier key, which is not shift and one also wants to hold and press other keys during this time, if used at all would probably be used for a button merging modifiers.
If we really want to not break this behaviour then I think the commit, which changes said default could be left out and we could make a note to change this in the next major release.

Reply:
One may or may not call this expected behaviour, as well—the button *does* say that it re-presses the modifier, after all :)

Reply:
Hmmm. Still the issue (?) with changing the default remains.

Reply:
Mh, I guess you're right that it wouldn't be super nice to change the default in a minor release. On the other hand, I don't think this will affect that many people, so it might be fine